README.md

```diff
@@ -20,7 +20,13 @@ on cQASM can be found in the Quantum Inspire
 
 ## Installation
 
-The source for the SDK can be found at Github. For the default installation execute:
+The Quantum Inspire SDK can be installed from PyPI via pip:
+
+```
+$ pip install quantuminspire
+```
+
+The source for the SDK can also be found at Github. For the default installation execute:
```

Comment:
Probably good to make clear that this is about installing from source by introducing a header before this line, `### Installing from source`. I would also include git clone instructions, something like:

```
$ git clone https://github.com/QuTech-Delft/quantuminspire
$ cd quantuminspire
$ pip install .
```